### PR TITLE
Feat/waitlist

### PR DESCRIPTION
**Pull request by annavaws**
## Description
Created a join waitlist endpoint that allows users to submit their email to join waitlist. You can access it in here `../v1/waitlist` 

### Request Body
Waitlist object example
```
{
  "email": "user@example.com",
}
```

### Response Body
Waitlist response body example
```
{
  "status_code": 200,
  "message": "Joined waitlist successfully"
}
```

## Response Statuses
- `200` SUCCESS Joined waitlist successfully
- `400` FAILED Email already exists in the waitlist
- `422` FAILED Invalid email address

### CODE REVIEW

**Comment by daffafaizan** [comment]
Can you provide the endpoint that we can hit in the description? It'll really help the people coming back to this merge request by making it easy to see what the endpoint is. Thanks!

**Comment by annavaws** [comment]
> Can you provide the endpoint that we can hit in the description? It'll really help the people coming back to this merge request by making it easy to see what the endpoint is. Thanks!

Good point! Done

**Review by daffafaizan — APPROVED** [review]
Code and changes look good.

File: alembic/versions/6f702f53e396_create_migrations.py
Comment (jeremyalv) [inline alembic/versions/6f702f53e396_create_migrations.py:1]: Let's remove this file from the Pull Request, as the changes are already included in the `feat/transcription-update` PR. I've added nullable=True

Comment (jeremyalv) [reply alembic/versions/6f702f53e396_create_migrations.py:1]: Plus some critical bugfixes were made there

**Review by jeremyalv — CHANGES_REQUESTED** [review]

File: alembic/versions/6f702f53e396_create_migrations.py
Comment (annavaws) [reply alembic/versions/6f702f53e396_create_migrations.py:1]: Okay, Done

**Review by jeremyalv — APPROVED** [review]

**Pull request merged**